controllers/nugus_controller/nugus_controller.cpp

```diff
@@ -677,16 +677,19 @@ class PlayerServer {
                 measurement->set_width(width);
                 measurement->set_height(height);
                 measurement->set_quality(-1);  // raw image (JPEG compression not yet supported)
-                const unsigned char* rgba_image = camera->getImage();
-                const int rgb_image_size        = width * height * 3;
-                unsigned char* rgb_image        = new unsigned char[rgb_image_size];
+                const unsigned char* bgra_image = camera->getImage();
+                const int rgba_image_size       = width * height * 4;  // RGBA has 4 channels
+                unsigned char* rgba_image       = new unsigned char[rgba_image_size];
+
                 for (int i = 0; i < width * height; i++) {
-                    rgb_image[3 * i]     = rgba_image[4 * i];
-                    rgb_image[3 * i + 1] = rgba_image[4 * i + 1];
-                    rgb_image[3 * i + 2] = rgba_image[4 * i + 2];
+                    rgba_image[4 * i + 0] = bgra_image[4 * i + 2];  // R
+                    rgba_image[4 * i + 1] = bgra_image[4 * i + 1];  // G
+                    rgba_image[4 * i + 2] = bgra_image[4 * i + 0];  // B
+                    rgba_image[4 * i + 3] = bgra_image[4 * i + 3];  // A
                 }
-                measurement->set_image(rgb_image, rgb_image_size);
-                delete[] rgb_image;
+
+                measurement->set_image(rgba_image, rgba_image_size);
```

Comment:
Does set_image expect this to change to 4 channels? How does it know?

Reply:
its a protobuf generated function and i believe it just stores the image as raw bytes where `rgba_image_size` specifies the size of the data in bytes, which is why we needed to change size calc from *3 to *4

Reply:
But what consumes the messages? It will have to change too?

Reply:
https://github.com/NUbots/NUbots/pull/1529 goes with this PR @KipHamiltons